source/sdk/lang/equalities.ooc

```diff
@@ -0,0 +1,47 @@
+// The following functions were moved from HashMap.ooc
```

Comment:
... and since there are no classes here, just global functions, that means you want the `e`, not `E`. Fine.